fix manpage-has-errors-from-man and hyphen-used-as-minus-sign

--- ./freqtweak.1.orig	2009-12-19 23:09:14.000000000 -0800
+++ ./freqtweak.1	2009-12-19 23:09:35.000000000 -0800
@@ -133,7 +133,7 @@
 Jack name. Default is freqtweak_1.
 .TP
 .B \-D <str>, \-\-tmpdir=<str>
-Jack server tmp directory (should match jackd --tmpdir).
+Jack server tmp directory (should match jackd \-\-tmpdir).
 .TP
 .B \-p <str>, \-\-preset=<str>
 Load the given preset initially.
@@ -149,7 +149,6 @@
 
 Start freqtweak first with this command line:
 
-.T
 .B freqtweak -n ft &
 
 Then start alsaplayer like so: 
